src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -83,6 +102,9 @@ public Map<String, String> getProperties(VariableResolver<String> resolver) {
                 if (m_sourceCodeOnly && getHasOptions()) {
                     properties.put(CoreConstants.SOURCE_CODE_ONLY, "");
                 }
+                if(m_scanSpeed!=null && !m_scanSpeed.equals("") && getHasOptions()){
```

Comment:
Instead of `.equals("")` can you use `.isEmpty()` ?

Reply:
Okay, sure.